ext/openssl/tests/openssl_get_cert_locations_basic.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+openssl_get_cert_locations() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+// openssl locations differ per distro.
+var_dump(count(openssl_get_cert_locations()));
+?>
+--EXPECTF--
+int(8)
```

Comment:
I admit the test is a bit silly, I'm not sure how a reasonable test can be written since the settings differ per installation.